specifications/api/swagger.yaml

```diff
@@ -5283,3 +5315,40 @@ paths:
           application/json:
             schema: *ApiSharedWalletPatchData
       responses: *responsesPatchSharedWallet
+
+  /shared-wallets/{walletId}/keys/{role}/{index}?hash=false:
+    get:
+      operationId: getSharedWalletKey
+      tags: ["Keys"]
```

Comment:
I think those two new endpoints should go to separate category "Shared Keys" grouped within Shared wallets as currently it renders:

![Screenshot from 2021-05-05 11-25-37](https://user-images.githubusercontent.com/42900201/117121792-64ee4000-ad95-11eb-8b59-cfd0195bcf49.png)

Reply:
fixed in c68a6bc.